markdown/dbaas.md

```diff
@@ -334,12 +402,58 @@ curl -X POST "http://localhost/v1/management/DBaaS/PXCClusters/GetCredentials" \
 
 API endpoint used in this step: [GetPXCClusterCredentials](ref:getpxcclustercredentials)
 
+### Create a PSMDB Cluster
+
+The PSMDB Create endpoint can also set defaults so, creating a PSMDB cluster can be made with a request like this:
+
+```bash
+curl -X POST "http://localhost/v1/management/DBaaS/PSMDBCluster/Create" \
+    -H "accept: application/json" \
+    -H "authorization: Basic YWRtaW46YWRtaW4=" \
+    -H "Content-Type: application/json" \
+    -d "{ \"kubernetes_cluster_name\": \"my_cluster\", \"expose\": true}"
+```
+
+#### Request fields
+
+```json
+{
+  "kubernetes_cluster_name": "string",
+  "name": "string",
+  "params": {
+    "cluster_size": 0,
+    "replicaset": {
+      "compute_resources": {
+        "cpu_m": 0,
+        "memory_bytes": "string"
+      },
+      "disk_size": "string"
+    },
+    "image": "string"
+  },
+  "expose": true
+}
+```
+
+| Field                                     | Description                           | Notes                                                        |
+| ----------------------------------------- | ------------------------------------- | ------------------------------------------------------------ |
+| kubernetes_cluster_name                   | Kubernetes cluster name               | Required                                                     |
+| name                                      | PSMDB cluster name                    | Default: `psmdb`+DB version+4 digits random number           |
+| cluster_size                              | Cluster size                          | Default: 3                                                   |
+| replicaset.compute_resources.cpu_m        | CPU resources millis                  | Default: 1000                                                |
+| replicaset.compute_resources.memory_bytes | Max memory size in bytes              | Default: 2 Gb                                                |
+| disk_size                                 | Max disk size                         | Default: 25 Gb                                               |
```

Comment:
The same as previous comment. 
In the new function that set the defaults in case the request hasn't these fields, we are setting the same values we have in the GUI for a small cluster.